k2/csrc/ragged_ops_inl.h

```diff
@@ -759,6 +759,129 @@ Array2<T> PadRagged(Ragged<T> &src, const std::string &mode, T padding_value) {
   return res;
 }
 
+// Prune a two axes ragged tensor on axis0
+template <typename T>
```

Comment:
This needs more documentation!
OK to just say this is a special case of PruneRagged with axis == 0, and refer to
the docs for PruneRagged in ragged_ops.h.